app/views/wiki/edit.html.erb

```diff
@@ -21,6 +21,13 @@
       <li><a href="/wiki/authoring-help#Advanced+formatting"><%= t('wiki.edit.advanced_formatting') %></a></li>
     </ul>
   </div>
+  <hr />
+  <div class="sidebar-nav hidden-sm">
+    <ul class="nav nav-list">
+      <li class="nav-header"><h4>Templates</h4></li>
+      <li><a href="/post">Method Template</a></li>
```

Comment:
>Clicking the link would go to the same URL as you're currently on, but with ?n=_____ appended!

 URL needs to be updated.

Reply:
I got that. But I have a doubt. We have the same template for 'Method Template' for all the node from 1 to 43. Just title and body of the node get changed on view different node by giving different values to param[:n]. You can also see this in `editor_controller#rich` method.
So I placed the URL for Method Template to `/post?n=1` and for lite it is `/wiki/new`. 
Please tell me if I am wrong or need to update something. Thanks :smile:

Reply:
I guess, there are some updates need to be done on the issue itself. In the issue, it's specified that we want to change the template of the wiki but `/post` is used for the creation of note so I am updating the issue.Also, @Ronaq13 you need to specify `n = 14552` as in production database node ID at which method template lite is there is at 14552. See here https://publiclab.org/wiki/wiki-templates .Also, it's `method template lite ` for which you need to append n=14552 and I  guess method template is the normal one which we use, so, for now, give it URL `/wiki/new`. @jywarren please check if I am right on details. Thanks.

Reply:
Done